<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="img/favicon.ico">

    <title>Christophe DJEMIEL Ph.D. looking for post-doc position in bioinformatics</title>

    <!-- Bootstrap core CSS -->
    <link href="css/bootstrap.min.css" rel="stylesheet">

    <!-- IE10 viewport hack for Surface/desktop Windows 8 bug -->
    <link href="css/ie10-viewport-bug-workaround.css" rel="stylesheet">

    <!-- Custom styles for this template -->
    <!-- <link href="css/navbar-fixed-top.css" rel="stylesheet"> -->
    <link href="css/sticky-footer-navbar.css" rel="stylesheet">
    <link href="css/custom-tricks.css" rel="stylesheet">
    <link href="css/bibbase.css" rel="stylesheet">
    <!-- <link href="font-awesome-4.6.3/css/font-awesome.min.css" rel="stylesheet"> -->
    <link href="fontawesome-free-5.0.6/web-fonts-with-css/css/fontawesome-all.min.css" rel="stylesheet">
    <!-- <link href="academicons-1.7.0/css/academicons.min.css" rel="stylesheet"> -->
    <link href="academicons-1.8.2/css/academicons.min.css" rel="stylesheet">
    <link href="flag-icon-css/css/flag-icon.min.css" rel="stylesheet">

    <!-- Just for debugging purposes. Don't actually copy these 2 lines! -->
    <!--[if lt IE 9]><script src="../../assets/js/ie8-responsive-file-warning.js"></script><![endif]-->
    <script src="js/ie-emulation-modes-warning.js"></script>

    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
    <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->
  </head>

  <body>

    <!-- Fixed navbar -->
    <nav class="navbar navbar-default navbar-fixed-top">
      <div class="container">
        <div class="navbar-header">
          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </button>
          <a class="navbar-brand" href="index.html">Christophe DJEMIEL</a>
        </div>
        <div id="navbar" class="navbar-collapse collapse">
          <ul class="nav navbar-nav">
            <li><a href="index.html">home</a></li>
            <li><a href="research.html">research</a></li>
            <li class="dropdown">
              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">previous work <span class="caret"></span></a>
              <ul class="dropdown-menu">
                <li class="dropdown-header">Master's degree</li>
                <li><a href="#">1<sup>st</sup> year</a></li>
                <li><a href="#">2<sup>nd</sup> year</a></li>
                <li><a href="#">Intership</a></li>
                <li role="separator" class="divider"></li>
                <li class="dropdown-header">iGEM competion</li>
                <li><a href="igem.html">Pseudo-bacterial eye</a></li>
              </ul>
            </li>
            <li><a href="softwares.html">softwares</a></li>
            <li class="active"><a href="cv.html">cv</a></li>
          </ul>
          <ul class="nav navbar-nav navbar-right social">
      		    <li><a href="https://github.com/cdjemiel" target="_blank" alt=""><i class="fab fa-github fa-lg"></i></a></li>
      		    <li><a href="https://www.researchgate.net/profile/Christophe_Djemiel" target="_blank" alt=""><i class="ai ai-lg ai-researchgate"></i></a></li>
      		    <li><a href="https://scholar.google.com/citations?user=Nm3Y4T0AAAAJ" target="_blank" alt=""><i class="ai ai-lg ai-google-scholar"></i></a></li>
    		      <li><a href="https://fr.linkedin.com/in/christophe-djemiel-6b80038a" target="_blank" alt=""><i class="fab fa-linkedin-in fa-lg"></i></a></li>
      		    <li><a href="https://www.mendeley.com/profiles/christophe-djemiel/" target="_blank" alt=""><i class="ai ai-lg ai-mendeley"></i></a></li>
    		    <li><a href="https://twitter.com/c_djemiel" target="_blank" alt=""><i class="fab fa-twitter fa-lg"></i></a></li>
    		  </ul>
        </div><!--/.nav-collapse -->
      </div>
    </nav>

    <div class="container">
		<div class="row">
			<div class="col-xs-9 col-sm-9 col-md-9">
				<div class="row">
					<div class="col-xs-12 col-sm-12 col-md-12">
						<h2>About</h2>
						<hr>
						<p class="text-justify" style="text-indent: 2em;">
							During my BTS internship in plant biotechnology, i quickly realized the importance that computing science would take in the coming years and i am strongly interested.
							<br>
							I went on biology license in the medecine university where i attended modules in plant biology and bioinformatics to then integrate the master in computer science in bordeaux. To validate my master i realize my intership on Genotoul platform where i developed various pipelines and visualization tools for metagenomics and i took a liking to manipulate the big data set.
							<br>
							Very oriented computer development, i decided to join the plant fiber team to better understand the key steps in microbial ecology of complex samples, from collect, sample preparation to data analysis and interpretation. I was able to draw upon my different skills molecular biology and bioinformatics.
							<br>
							I defend my Ph.D. thesis in 20<sup>th</sup> December 2017 and I'm looking for a postdoc position in the continuity of my thesis in meta-omics.
						</p>
					</div>
				</div>
				<div class="row">
					<div class="col-xs-12 col-sm-12 col-md-12">
						<h2>Professional experience</h2>
						<hr>
						<div class="row">
							<div class="col-xs-2 col-sm-2 col-md-2">
								<span class="label label-default">2013 - 2017</span>
								<br>
								<span class="label label-success">UGSF (Fr)</span>
							</div>
							<div class="col-xs-10 col-sm-10 col-md-10">
								<strong class="text-success">Ph.D. thesis in Environmental Biology, Population and Ecology</strong>
								<br>
								<strong class="text-muted">Study of bacterial and fungal diversity during flax dew-retting by using metabarcoding</strong>
								<br>
								Surpervisors: D<sup>r</sup> <a href="https://www.researchgate.net/profile/Sebastien_Grec" target="_blank" class="text-orange">Sébastien Grec</a> and P<sup>r</sup> <a href="https://www.researchgate.net/profile/Simon_Hawkins" target="_blank" class="text-orange">Simon Hawkins</a>
								<ul>
									<li>Preparation of samples from field experimental design to Illumina sequencing.
									<li>Re-design universal reverse primer of V3-V4 rRNA gene subregion for plant-associated bacteria studies.
									<li>Development of various methods to remove rRNA or enrich to mRNA from plant-associated microbiome for metatranscriptomic studies.
									<li>Implementation of pipelines and statistical analysis for microbial divertsity.
									<li>Looking for microbial and CAZymes markers.
								</ul>
							</div>
						</div>
						<div class="row">
							<div class="col-xs-2 col-sm-2 col-md-2">
								<span class="label label-default">2013 (6 months)</span>
								<br>
								<span class="label label-success">Genotoul (Fr)</span>
							</div>
							<div class="col-xs-10 col-sm-10 col-md-10">
								<strong class="text-success">Master’s intership in computer science</strong>
								<br>
								<strong class="text-muted">Development of metabarcoding pipelines for classify and clutering 16S rDNA</strong>
								<br>
								Supervisor: <a href="https://www.researchgate.net/profile/Jerome_Mariette" target="_blank" class="text-orange">Jérôme Mariette</a> - Software engineer (<a href="http://bioinfo.genotoul.fr/" target="_blank" class="text-orange">Genotoul bioinfo</a>, INRA, Toulouse - France)
								<ul>
									<li>Development pipelines (Roche 454 and Illumina Miseq) with <a href="http://genoweb.toulouse.inra.fr:8090/app/index.html" target="_blank" class="text-orange">Jflow</a> API based on <a href="http://ccl.cse.nd.edu/software/makeflow/" target="_blank" class="text-orange">Makeflow</a> and <a href="https://bitbucket.org/pbui/weaver" target="_blank" class="text-orange">Weaver</a> and be integrated in <a href="https://mulcyber.toulouse.inra.fr/projects/ngspipelines/" target="_blank" class="text-orange">NGSpipelines</a> and <a href="http://ng6.toulouse.inra.fr/" target="_blank" class="text-orange">NG6</a>
									<li>Implementation Perl script to convert taxonomy file from mothur to hierarchical <a href="https://github.com/marbl/Krona/wiki" target="_blank" class="text-orange">Krona</a>
									<li>Implementaion of jQuery plugin for comparing lists with Venn Diagrams: <a href="http://bioinfo.genotoul.fr/jvenn/index.html" target="_blank" class="text-orange">jvenn</a>
								</ul>
							</div>
						</div>
						<div class="row">
							<div class="col-xs-2 col-sm-2 col-md-2">
								<span class="label label-default">2011 - 2013</span>
								<br>
								<span class="label label-success">iGEM (Fr)</span>
							</div>
							<div class="col-xs-10 col-sm-10 col-md-10">
								<strong class="text-success">International Genetically Engineered Machine (iGEM) Competition</strong>
								<br>
								<strong class="text-muted">Pseudo-bacterial eye</strong>
								<br>
								Supervisors: D<sup>r</sup> <a href="https://www.researchgate.net/researcher/38792622_Denis_Dupuy" target="_blank" class="text-orange">Denis Dupuy</a> (Genetics and molecular biology) and D<sup>re</sup> <a href="http://dept-info.labri.fr/~beurton/" target="_blank" class="text-orange">Marie Beurton-Aimar</a> (Computer science)
								<ul>
									<li>Involvement in setting up of the first team of Bordeaux
									<li>Responsible for modeling and simulation of complex existing regulatory mechanisms in eukaryotic in prokaryotes organisms. Creating a Python script language for modeling biological patterns.
									<li>Development of a wiki website for <a href="http://2012.igem.org/Team:Bordeaux" target="_blank" class="text-orange">your project</a>
									<li>Participation of oral and poster communication of our project in Amsterdam, 5<sup>th</sup> - 7<sup>th</sup> October 2012
									<li>The Bordeaux team won a bronze medal
								</ul>
							</div>
						</div>
						<div class="row">
							<div class="col-xs-2 col-sm-2 col-md-2">
								<span class="label label-default">2008 - 2009 (4 months)</span>
								<br>
								<span class="label label-success">INRA (Fr)</span>
							</div>
							<div class="col-xs-10 col-sm-10 col-md-10">
								<strong class="text-success">BTS intership with highest honors</strong>
								<br>
								<strong class="text-muted">Functional analysis of cell cycle genes regulator and endocycle from tomato</strong>
								<br>
								Supervisors: D<sup>re</sup> <a href="http://glycomev.univ-rouen.fr/CV-French-English/Elodie.html" target="_blank" class="text-orange">Elodie Mathieu-Rivet</a> (Glyco-MEV, University of Rouen - France) and P<sup>r</sup> <a href="https://www.researchgate.net/profile/Michel_Hernould" target="_blank" class="text-orange">Michel Hernould</a> (UMR 1332 Biologie du Fruit et Pathologie - INRA / University of Bordeaux - France)
								<ul>
									<li>Constructing various expression vectors of cell cycle genes for a transitoire approach by biolistic method and for a stable approach by <i>Agrobacterium tumefaciens</i>
									<li>Development of plasmid manager tool (VBA langage) called "Plasmidator" on my free time
								</ul>
							</div>
						</div>
					</div>
				</div>
				<div class="row">
					<div class="col-xs-12 col-sm-12 col-md-12">
						<h2>Education and diplomas</h2>
						<hr>
						<ul>
							<li><span class="label label-default">2017</span>&nbsp;<strong>Ph.D. thesis in Environmental Biology, Population and Ecology</strong>, <a href="http://ugsf-umr-glycobiologie.univ-lille1.fr/" target="_blank" class="text-orange">UGSF laboratory</a>, University of Lille 1 (France), with highest honors</li>
							<li><span class="label label-default">2013</span>&nbsp;<strong>Master's degree in computer science</strong>, <a href="http://www.u-bordeaux.com/" target="_blank" class="text-orange">University of Bordeaux</a> (France), with highest honors</li>
							<li><span class="label label-default">2011</span>&nbsp;<strong>Bachelor's degree in Biology</strong>, <a href="http://www.u-bordeaux.com/" target="_blank" class="text-orange">University of Bordeaux</a> (France)</li>
							<li><span class="label label-default">2009</span>&nbsp;<strong>2-year technical college degree</strong>, <a href="http://www.lyceesaintlouis.fr/Joomla/index.php/formations/bts/biotechnologies" target="_blank" class="text-orange">St. Louis High School</a> (Bordeaux - France) with honors</li>
							<li><span class="label label-default">2007</span>&nbsp;<strong>French HS diploma in Science major</strong>, <a href="http://www.lycee-montesquieu.fr/" target="_blank" class="text-orange">Montesquieu High School</a> (Bordeaux - France)</li>

						</ul>
					</div>
				</div>
				<div class="row">
					<div class="col-xs-12 col-sm-12 col-md-12">
						<h2>Teaching</h2>
						<hr>
						<p class="text-justify" style="text-indent: 2em;">Generally, I supervised a group of about 30 students for the practical work. I was in charge the elaboration of quick quiz at beginning of lessons, the evaluation of scientific experiments and the correction of drawings. For the "Initiation of computer science" course for a Next Generation Sequencing worshop for 15 M.Sc. students, I developed a lesson (duration 6h) with pratical work.</p>
						<table class="table">
						  <tr>
						    <thead>
						      <th>Period</th>
						      <th>Institution</th>
						      <th>Level</th>
						      <th>Module</th>
						      <th>Type</th>
						      <th>Duration</th>
						    </thead>
						  </tr>
						  <tr>
						    <td rowspan="3">2015 - 2016</td>
						    <td><a href="http://www.univ-lille1.fr" target="_blank" class="text-orange">University of Lille 1</a></td>
						    <td>1st year of Bachelor degree</td>
						    <td>Plant biology</td>
						    <td>Pratical work</td>
						    <td>16h</td>
						  </tr>
						  <tr>
						    <td><a href="http://www.univ-lille1.fr" target="_blank" class="text-orange">University of Lille 1</a></td>
						    <td>M.Sc. in Biotechnology</td>
						    <td>Initiation of computer science</td>
						    <td>Teaching assistant</td>
						    <td>4h</td>
						  </tr>
						  <tr>
						    <td><a href="http://www.univ-lille1.fr" target="_blank" class="text-orange">University of Lille 1</a></td>
						    <td>1st year of Bachelor degree</td>
						    <td>Cellular biology</td>
						    <td>Pratical work</td>
						    <td>45h</td>
						  </tr>
						  <tr>
						    <td rowspan="2">2014 - 2015</td>
						    <td><a href="http://www.univ-lille1.fr" target="_blank" class="text-orange">University of Lille 1</a></td>
						    <td>1st year of Bachelor degree</td>
						    <td>Plant biology</td>
						    <td>Pratical work</td>
						    <td>58h</td>
						  </tr>
						  <tr>
						    <td><a href="http://www.univ-lille1.fr" target="_blank" class="text-orange">University of Lille 1</a></td>
						    <td>M.Sc. in Biotechnology</td>
						    <td>Initiation of computer science</td>
						    <td>Teaching assistant</td>
						    <td>6h</td>
						  </tr>
						  <tr>
						    <td>2015 - 2016</td>
						    <td><a href="http://www.univ-lille1.fr" target="_blank" class="text-orange">University of Lille 1</a></td>
						    <td>1st year of Bachelor degree</td>
						    <td>Plant biology</td>
						    <td>Pratical work</td>
						    <td>64h</td>
						  </tr>
						  <tr>
						    <td rowspan="2" colspan="4"><strong>Total</strong></td>
						    <td><strong>Pratical work</strong></td>
						    <td><strong>183h</strong></td>
						  </tr>
						  <tr>
						    <td><strong>Teaching assistant</strong></td>
						    <td><strong>10h</strong></td>
						  </tr>
						</table>
					</div>
				</div>
        <div class="row">
          <div class="col-xs-12 col-sm-12 col-md-12">
            <h2> Fellowships, Grants and Awards</h2>
            <hr>
            <h3 class="text-muted">Awards</h3>
            <hr>
            <ul>
              <li><span class="label label-default">2017</span>&nbsp;<strong>Best poster price RFP</strong>, <a href="https://journees.inra.fr/jrfp2017/" target="_blank" class="text-orange">Results</a>, Seminar Réseau Français des Parois, Orléans (Fr)</li>
              <li><span class="label label-default">2012</span>&nbsp;<strong>Bronze medal Europe Jamboree For iGEM 2012 Championship</strong>, <a href="http://igem.org/Results?year=2012" target="_blank" class="text-orange">Results</a>, The International Genetically Engineered Machine, Amsterdam (Nl)</li>
            </ul>
          </div>
        </div>
				<div class="row">
					<div class="col-xs-12 col-sm-12 col-md-12" id="publication-text-orange">
						<h2>Publications</h2>
						<hr>
						<h3 class="text-muted">Articles in international peer-reviewed scientific journals</h3>
						<p class="bg-warning"><strong>Citations: 146, H-index:1 based on Google Scholar (May 2018)</strong></p>
            <div class="embed-responsive embed-responsive-16by9">
            <iframe class="embed-responsive-item" src="http://bibbase.org/service/mendeley/5d5a4700-7a44-3df0-8715-150fe9bdba92"></iframe>
            </div>
					</div>
				</div>
				<div class="row">
					<div class="col-xs-12 col-sm-12 col-md-12">
						<h2>Conferences and oral communications</h2>
						<hr>
            <h3 class="text-muted">Invited Talks</h3>
            <hr>
            <div class="media">
              <div class="media-left media-middle">
                <a href="#" target="_blank">
                  <img class="media-object" width="64" height="64" src="img/logo_fare.jpg" alt="Seminar - UGSF">
                </a>
              </div>
              <div class="media-body">
              <h4 class="media-heading">Étude  de la dynamique des communautés microbiennes au  cours  du rouissage du lin par <i>metabarcoding</i></h4>
              <h5>Christophe Djemiel</h5>
              <span class="label label-default">FARE Seminar</span>
              &nbsp;<span class="flag-icon flag-icon-fr" title="Lille - France"></span>&nbsp;INRA Reims - France&nbsp;<i class="fas fa-calendar-check fa-fw" aria-hidden="true"></i>09<sup>th</sup> of February 2018&nbsp;<i class="fas fa-clock fa-fw" aria-hidden="true"></i> 45min
              &nbsp;<a class="headingCV text-orange">abstract <i class="fa fa-caret-down" aria-hidden="true"></i></a>
              <div class="alert alert-success contentCV" role="alert">
                <p class="text-justify">
                Coming soon ...
                </p>
              </div>
              </div>
            </div>
            <div class="media">
              <div class="media-left media-middle">
                <a href="#" target="_blank">
                  <img class="media-object" width="64" height="64" src="img/ulco.jpg" alt="Seminar - UGSF">
                </a>
              </div>
              <div class="media-body">
              <h4 class="media-heading">Les méta-omiques au service de l'étude des écosystèmes microbiens</h4>
              <h5>Christophe Djemiel</h5>
              <span class="label label-default">Conférence sur l'appui des techniques "omiques"</span>
              &nbsp;<span class="flag-icon flag-icon-fr" title="Lille - France"></span>&nbsp;ULCO - France&nbsp;<i class="fas fa-calendar-check fa-fw" aria-hidden="true"></i>08<sup>th</sup> of February 2018&nbsp;<i class="fas fa-clock fa-fw" aria-hidden="true"></i> 45min
              &nbsp;<a class="headingCV text-orange">abstract <i class="fa fa-caret-down" aria-hidden="true"></i></a>
              <div class="alert alert-success contentCV" role="alert">
                <p class="text-justify">
                Coming soon ...
                </p>
              </div>
              </div>
            </div>
						<h3 class="text-muted">International conference with proceedings</h3>
						<hr>
						<div class="media">
						  <div class="media-left media-middle">
						    <a href="http://www.plant-phenotyping.org/home_of_2nd_meeting" target="_blank">
						      <img class="media-object" width="64" height="64" src="img/logo_cost.png" alt="2nd general COST MEETING">
						    </a>
						  </div>
						  <div class="media-body">
						    <h4 class="media-heading">Phenotyping microbial diversity during flax dew-retting by using targeted-metagenomics</h4>
							<h5><u>Christophe Djemiel</u>, Sébastien Grec, and Simon Hawkins</h5>
							<span class="label label-default">2<sup>nd</sup> general COST MEETING</span>
							&nbsp;<span class="flag-icon flag-icon-dk" title="Copenhagen - Denmark"></span>&nbsp;Copenhagen - Denmark&nbsp;<i class="fas fa-calendar-check fa-fw" aria-hidden="true"></i>18<sup>th</sup> - 20<sup>th</sup> of April 2016&nbsp;<i class="fas fa-clock fa-fw" aria-hidden="true"></i> 20min
							&nbsp;<i class="fas fa-file-pdf fa-fw" aria-hidden="true"></i>&nbsp;<a href="http://www.plant-phenotyping.org/lw_resource/datapool/_items/item_550/abstractbookphenomenallcph2016.pdf" target="_blank" class="text-orange">Proceedings</a>
							&nbsp;<a class="headingCV text-orange">abstract <i class="fa fa-caret-down" aria-hidden="true"></i></a>
							<div class="alert alert-success contentCV" role="alert">
							  <p class="text-justify">
								  Flax (Linum usitatissimum L.) is an economically important oil and fiber crop. Flax
								  bast fibers, rich in cellulose, are traditionally used to produce textiles (linen) and can also be
								  used for reinforcing composite polymers as an environmentally friendly alternative to glass
								  fibers [1]. The ‘quality’ of both textiles and composite materials is influenced by the ‘quality’
								  of the plant fibers that arrive in the factory. Plant fiber quality itself is related to fiber cell wall
								  structure determined during plant growth, and modified to a greater or lesser extent during
								  fiber extraction.
								  The first step in the industrial transformation of flax stems into fibers is achieved by
								  dew-retting on the soil surface. At harvest, the flax stems are ‘pulled’ (up-rooted) and laid
								  down directly on the soil in swaths (strips). Subsequent exposure to alternating periods of rain
								  and heat combined with the development and the action of soil microflora favor the separation
								  of cellulosic bast fibers from the stems via the hydrolysis of cell wall polymers (hemicelluloses,
								  pectins). Despite many studies of this process [2,3,4] relatively little is known about i)
								  the composition of the microflora population, ii) the kinetics of microbial colonization of
								  plant material and iii) the evolution of the microbial population in the soil. Soil retting remains
								  a ‘black box’ (very few microbial species have so far been identified) and our current
								  lack of knowledge of this process represents a major obstacle to a more efficient retting process.
								  In the framework of a collaborative French ‘Future project’ ‘StructuratIoN de la filière
								  Fibres techniques d’OrigiNe végétale pour usages matérIaux’ (SINFONI) we are conducting a
								  targeted-metagenomics approach to phenotype for the first time the composition of the microbial
								  communities (focusing on bacteria and fungi) involved in flax dew-retting. This approach
								  has also allowed us to identify some potential major enzymatic functions related to cell wall
								  degradation based on functional prediction using the bioinformatic software PICRUSt.
								  In this communication we will briefly present the methodology used to explore microbial
								  diversity using the Next Generation of Sequencing (Illumina MiSeq system) and explain
								  how this data is helping us to understand the impact of plant cell wall degrading enzymes on
								  flax stems. The range of identified biological markers should contribute to improving the extraction
								  process and hence fiber quality.
								  <br>
								  [1] Baley, C. (2002). Analysis of the flax fibres tensile behaviour and analysis of the tensile stiffness increase. Composites Part A: Applied
								  Science and Manufacturing, 33(7), 939–948. http://doi.org/10.1016/S1359-835X(02)00040-4
								  <br>
								  [2] Akin, D. E. (2013). Linen most useful: Perspectives on structure, chemistry, and enzymes for retting flax. ISRN Biotechnology, 2013, 23.
								  http://doi.org/http://dx.doi.org/10.5402/2013/186534
								  <br>
								  [3] Henriksson, G., & Akin, D. (1997). Identification and retting efficiencies of fungi isolated from dew-retted flax in the United States and
								  europe. Applied and Environmental Microbiology. Retrieved from http://aem.asm.org/content/63/10/3950.short
								  <br>
								  [4] Martin, N., Mouret, N., Davies, P., & Baley, C. (2013). Influence of the degree of retting of flax fibers on the tensile properties of single fibers and short fiber/polypropylene composites. Industrial Crops and Products, 49, 755–767. http://doi.org/10.1016/j.indcrop.2013.06.012
							</div>
							</p>
						  </div>
						</div>
						<div class="media">
						  <div class="media-left media-middle">
						    <a href="http://lignobiotech.es/" target="_blank">
						      <img class="media-object" width="64" height="64" src="img/logo_LIGNOBIO.png" alt="4th Symposium on Biotechnology applied to Lignocelluloses">
						    </a>
						  </div>
						  <div class="media-body">
						    <h4 class="media-heading">Tracking dynamics of ax reeting by evolution in stem architecture, call wall structure, and biological activities</h4>
							<h5>Brigitte Chabbert, Christophe Djemiel, Antoine Portelette, Jordane Ossemond, Sébastien Grec, Simon Hawkins, <u>Bernard Kurek</u></h5>
							<span class="label label-default">LIGNO BIOTECH IV</span>
							&nbsp;<span class="flag-icon flag-icon-es" title="Madrid - Spain"></span>&nbsp;Madrid - Spain&nbsp;<i class="fas fa-calendar-check fa-fw" aria-hidden="true"></i>19<sup>th</sup> - 22<sup>th</sup> of June 2016&nbsp;<i class="fas fa-clock fa-fw" aria-hidden="true"></i> 20min
							&nbsp;<i class="fas fa-file-pdf fa-fw" aria-hidden="true"></i>&nbsp;<a href="#" target="_blank" class="text-orange">Proceedings</a>
							&nbsp;<a class="headingCV text-orange">abstract <i class="fas fa-caret-down" aria-hidden="true"></i></a>
							<div class="alert alert-success contentCV" role="alert">
							  <p class="text-justify">
								  The use of plant fibers has spread to new applications intended to replace synthetic fibers for natural fibers [1]. Among plant biomass, flax or hemp are economically important fiber crops owing to their long fibers that are characterized by a very thick secondary cell wall containing high amounts of crystalline cellulose with high tensile strength. These bast fibers are distributed as bundles in the outer tissue of the stem.
								  Retting is a partial decomposition performed on soil after harvest. This process promotes the growth of soil microorganisms that degrade pectins and hemicelluloses responsible for the cohesion between fibers, thus facilitating their mechanical extraction and improving the quality of the fibers [2]. However field retting is mainly based on empirical knowledge which makes it difficult to control [3].Since different industrial processes require a regular supply of uniform, high quality fibers, detailed understanding on the retting taking place in the field could provide indicators for improved management of the process. However there are few data in the literature on the dynamic of fiber crops structures during field retting and the composition of the microflora population associated with the process [4, 5].
								  We have undertaken a dynamic approach on flax retting while combining 1) chemical, biological and microscopic characterizations of the straw collected on the field at different periods from harvesting stage, and 2) targeted-metagenomics approach to determine the composition of the microbial communities (bacteria and fungi) involved in flax dew-retting. Main changes in the stem structure and enzymatic activities were shown during the first weeks of dew-retting in good agreement with rapid microbial colonization of the bark tissues.
								  <br>
								  [1] Madsen B, Gamstedt EK. Wood versus Plant Fibers: Similarities and Differences in Composite Applications. Advances in Materials Science and Engineering 2013. 564346.
								  <br>
								  [2] Sharma HS, Van Sumere CF. The Biology and Processing of Flax. M Publications: Belfast, 1992. 576 p.
								  <br>
								  [3] Tahir PM, Ahmed AB, Saiful Azry SOA, Ahmed Z. Retting process of some bast plant fibres ans its effect on fibre quality: a review. Bioresources 2011, 6, 5260-528.
								  <br>
								  [4] Liu M, Fernando D, Daniel G, Madsen B, Meyer AS, Ale MT, Thygesen A. Industrial Crops and Products 2015, 69, 29-39.
								  <br>
								  [5] Ribeiro A, Pochart P, Day A, Mennuni S, Bono P, Baret JL, Spadoni JL, Mangin I. Applied Microbiology and Biotechnology 2015, 99 (10), 4471-4484.
                </p>
              </div>
						  </div>
						</div>
						<h3 class="text-muted">National conference with proceedings</h3>
						<hr>
						<div class="media">
						  <div class="media-left media-middle">
						    <a href="http://multihemp.eu/" target="_blank">
						      <img class="media-object" width="64" height="64" src="img/logo_multihemp.png" alt="Multihemp workshop">
						    </a>
						  </div>
						  <div class="media-body">
						    <h4 class="media-heading">Microbial diversity of retting - From cultured to uncultured genome sequences</h4>
							<h5>Christophe Djemiel, Sébastien Grec, and Simon Hawkins</h5>
							<span class="label label-default">Multihemp workshop</span>
							&nbsp;<span class="flag-icon flag-icon-fr" title="Lille - France"></span>&nbsp;Lille - France&nbsp;<i class="fas fa-calendar-check fa-fw" aria-hidden="true"></i>18<sup>th</sup> of January 2016&nbsp;<i class="fas fa-clock fa-fw" aria-hidden="true"></i> 15min
							&nbsp;<i class="fas fa-file-pdf fa-fw" aria-hidden="true"></i>&nbsp;<a href="pdf/FIBRE_QUALITY_WORKSHOP_Lille_January_2016.pdf" target="_blank" class="text-orange">Proceedings</a>
							&nbsp;<a class="headingCV text-orange">abstract <i class="fa fa-caret-down" aria-hidden="true"></i></a>
							<div class="alert alert-success contentCV" role="alert">
							  <p class="text-justify">
							  Coming soon ...
							  </p>
							</div>
						  </div>
						</div>
						<div class="media">
						  <div class="media-left media-middle">
						    <a href="#" target="_blank">
						      <img class="media-object" width="64" height="64" src="img/flax_flower.jpg" alt="Transatlantic Flax Research meeting">
						    </a>
						  </div>
						  <div class="media-body">
						    <h4 class="media-heading">Dynamics of bacterial and fungal communities during the dew-retting of flax: Metagenomics approaches</h4>
							<h5>Christophe Djemiel, Sébastien Grec, and Simon Hawkins</h5>
							<span class="label label-default">Transatlantic Flax Research meeting</span>
							&nbsp;<span class="flag-icon flag-icon-fr" title="Lille - France"></span>&nbsp;Lille - France&nbsp;<i class="fas fa-calendar-check fa-fw" aria-hidden="true"></i>9<sup>th</sup> of June 2015&nbsp;<i class="fas fa-clock fa-fw" aria-hidden="true"></i> 20min
							&nbsp;<i class="fas fa-file-pdf fa-fw" aria-hidden="true"></i>&nbsp;<a href="pdf/Transatlantic_Flax_research_Agenda.pdf" target="_blank" class="text-orange">Proceedings</a>
							&nbsp;<a class="headingCV text-orange">abstract <i class="fa fa-caret-down" aria-hidden="true"></i></a>
							<div class="alert alert-success contentCV" role="alert">
							  <p class="text-justify">
							  Coming soon ...
							  </p>
							</div>
						  </div>
						</div>
						<h3 class="text-muted">Seminar and oral communications without proceedings</h3>
						<hr>
						<div class="media">
						  <div class="media-left media-middle">
						    <a href="#" target="_blank">
						      <img class="media-object" width="64" height="64" src="img/logo_UGSF.jpg" alt="Seminar - UGSF">
						    </a>
						  </div>
						  <div class="media-body">
						    <h4 class="media-heading">Étude de la dégradation pariétale au cours du rouissage du lin : Approches par méta-génomique fonctionnelle et diversité taxonomique</h4>
							<h5>Christophe Djemiel, Sébastien Grec, and Simon Hawkins</h5>
							<span class="label label-default">UGSF seminar</span>
							&nbsp;<span class="flag-icon flag-icon-fr" title="Lille - France"></span>&nbsp;Lille - France&nbsp;<i class="fas fa-calendar-check fa-fw" aria-hidden="true"></i>14<sup>th</sup> of March 2014&nbsp;<i class="fas fa-clock fa-fw" aria-hidden="true"></i> 45min
							&nbsp;<a class="headingCV text-orange">abstract <i class="fas fa-caret-down" aria-hidden="true"></i></a>
							<div class="alert alert-success contentCV" role="alert">
							  <p class="text-justify">
							  Coming soon ...
							  </p>
							</div>
						  </div>
						</div>
						<div class="row">
							<div class="col-xs-12 col-sm-12 col-md-12">
								<h2>Posters</h2>
								<hr>
                <div class="media">
                  <div class="media-left media-middle">
                    <a href="https://journees.inra.fr/jrfp2017/" target="_blank">
                      <img class="media-object" width="64" height="64" src="img/logo_RFP.jpg" alt="10e Journées du Réseau Français des Parois">
                    </a>
                  </div>
                  <div class="media-body">
                    <h4 class="media-heading">Structural variability in the caulosphere and rhizosphere microbiome during flax dew-retting</h4>
                  <h5><u>Christophe Djemiel</u>, Sébastien Grec, and Simon Hawkins</h5>
                  <span class="label label-default">11th RFP</span>
                  &nbsp;<span class="flag-icon flag-icon-fr" title="Orléans - France"></span>&nbsp;Orléans - France&nbsp;<i class="fas fa-calendar-check fa-fw" aria-hidden="true"></i>27<sup>th</sup> - 29<sup>th</sup> of June 2017&nbsp;<i class="fas fa-clock fa-fw" aria-hidden="true"></i> 3min
                  &nbsp;<i class="fas fa-file-pdf fa-fw" aria-hidden="true"></i>&nbsp;<a href="pdf/2017_rfp_orléans.pdf" target="_blank" class="text-orange">Proceedings</a>
                  &nbsp;<a class="headingCV text-orange">abstract <i class="fas fa-caret-down" aria-hidden="true"></i></a>
                  <div class="alert alert-success contentCV" role="alert">
                    <p class="text-justify">
                    The existence of spun and colored flax fibers dating from the Upper Paleolithic suggests that man has long exploited the biotechnological process known as retting to extract these fibers from the flax plant for textile production [1, 2]. In the case of dew-retting, the complex microbiological process is achieved directly on the soil surface of the field [3, 4]. Despite many studies of this process to evaluate the degree of retting [5], relatively little is known about (i) the composition and the dynamics of the microflora population during retting, (ii) the spatial-temporal effect on the microbiome and (iii) the composition and the evolution of Carbohydrate Active enZymes degrading plant cell walls [6].
                    To improve our understanding of dew-retting, we used a high throughput sequencing (HTS) metabarcoding approach (16S rRNA and ITS regions, Illumina Miseq system) to analyze the composition and dynamics of the rhizosphere (soil) and caulosphere (stem) microbial populations during this process.
                    This study was conducted within the framework of the collaborative French ‘Future project’ ‘StructuratIoN de la filière Fibres techniques d’OrigiNe végétale pour usages matérIaux’ (SINFONI). Our results allowed us to (i) report the first exhaustive HTS microbial inventory focusing on both bacteria (16S rRNA) and fungi (Internal Transcript Spacer 2), (ii) evaluate the impact of a number of different parameters likely to affect microbiome structural variability and retting efficiency and (iii) to ascertain the bacterial and fungal groups responsible for the observed community differentiation in the caulosphere. In addition, we also used PICRUSt to predict potential bacterial enzymatic functions related to carbohydrate degradation and FUNGuild to determine fungal trophic modes [7, 8].
                    <br>
                    [1] Gübitz, G.M. & Cavaco-Paulo (2001) Journal of biotechnology, 89(2–3), pp.89–90.
                    <br>
                    [2] Kvavadze, E. et al. (2009) Science, 325(5946), pp.1359–1359
                    <br>
                    [3] Md. Tahir, P. et al. (2011) BioResources, 6(4), pp.5260–5281
                    <br>
                    [4] Akin, D.E., (2013) ISRN Biotechnology, 2013, p.23
                    <br>
                    [5] Martin, N. et al. (2013) Industrial Crops and Products, 49, pp.755–767
                    <br>
                    [6] Lombard, V. et al. (2014) Nucleic acids research, 42(Database issue), pp.D490-5 [7] Langille, M.G. et al. (2013) Nat Biotechnol, 31(9), pp.814–821
                    <br>
                    [8] Nguyen, N.H. et al. (2016) Fungal Ecology, 20, pp.241–248.
                    </p>
                  </div>
                  </div>
                </div>

                <div class="media">
                  <div class="media-left media-middle">
                    <a href="https://project.inria.fr/jobim2017/fr/" target="_blank">
                      <img class="media-object" width="64" height="64" src="img/logo_jobim2017.png" alt="18ème édition des Journées Ouvertes en Biologie, Informatique et Mathématiques">
                    </a>
                  </div>
                <div class="media-body">
                  <h4 class="media-heading">Microbial diversity and plant cell wall-degrading enzyme dynamics during dew-retting of flax – one of the oldest applications of biotechnology to textile</h4>
                <h5><u>Christophe Djemiel</u>, Sébastien Grec, and Simon Hawkins</h5>
                <span class="label label-default">18th JOBIM</span>
                &nbsp;<span class="flag-icon flag-icon-fr" title="Lille - France"></span>&nbsp;Lille - France&nbsp;<i class="fas fa-calendar-check fa-fw" aria-hidden="true"></i>3<sup>th</sup> - 6<sup>th</sup> of July 2017&nbsp;<i class="fas fa-clock fa-fw" aria-hidden="true"></i> 3min
                &nbsp;<i class="fas fa-file-pdf fa-fw" aria-hidden="true"></i>&nbsp;<a href="pdf/2017-jobim-actes.pdf" target="_blank" class="text-orange">Proceedings</a>
                &nbsp;<a class="headingCV text-orange">abstract <i class="fa fa-caret-down" aria-hidden="true"></i></a>
                <div class="alert alert-success contentCV" role="alert">
                  <p class="text-justify">
                    The existence of spun and colored flax fibers dating from the Upper Paleolithic suggests that man has long exploited the biotechnological natural process known as dew-retting to extract these fibers from the flax plant for textile production [1,2]. This process is achieved directly on the soil surface of the field [3,4]. Despite many studies of this process to evaluate the degree of retting [5], relatively little is known about (i) the composition and the evolution of the microflora population during retting, (ii) the kinetics of the microbial communities colonizing the plant material and (iii) the composition and the evolution of Carbohydrate Active enZymes degrading plant cell walls [6].
                    To improve our understanding of dew-retting, we first used a metabarcoding approach to identify the membership and structure of the microbial communities (focusing on bacteria and fungi). This approach also allowed us to identify (i) some potential bacterial major enzymatic functions related to carbohydrate degradation based on functional prediction using PICRUSt (http://picrust.github.io/picrust/) and (ii) a strong pattern of fungal trophic modes [7,8]. In a second step we developed a metatranscriptomic approach to access of the evolution of the exogenous (soil) and endogenous (plant) enzymatic arsenal potentially across the Tree of Life, involved in the degradation of carbohydrate and aromatic substances in decomposing plant matter.
                    The methodology used to explore microbial and enzymatic diversity using High Throughput Sequencing (Illumina system) will be described. We will then correlate colonization complexity dynamics to the progress in plant cell wall degradation. Finally, the microbial ecology of the retting process will be compared to other natural plant material (e.g. forest litter) degradation processes.
                    <br>
                    Acknowledgements
                    <br>
                    This work is funded within the framework of the collaborative French ‘Future project’ SINFONI.
                    <br>
                    References
                    <br>
                    [1] Roland, J.-C., Mosiniak, M. & Roland, D., 1995. Dynamique du positionnement de la cellulose dans les parois des fibres textiles du lin ( Linum usitatissimum ). Acta Botanica Gallica, 142(5), pp.463–484.
                    <br>
                    [2] Kvavadze, E. et al., 2009. 30,000-Year-Old Wild Flax Fibers. Science, 325(5946), pp.1359–1359.
                    <br>
                    [3] Md. Tahir, P. et al., 2011. Retting process of some bast plant fibres and its effect on fibre quality: A review. BioResources, 6(4), pp.5260–5281.
                    <br>
                    [4] Akin, D.E., 2013. Linen most useful: Perspectives on structure, chemistry, and enzymes for retting flax. ISRN Biotechnology, 2013, p.23.
                    <br>
                    [5] Martin, N. et al., 2013. Influence of the degree of retting of flax fibers on the tensile properties of single fibers and short fiber/polypropylene composites. Industrial Crops and Products, 49, pp.755–767.
                    <br>
                    [6] Lombard, V. et al., 2014. The carbohydrate-active enzymes database (CAZy) in 2013. Nucleic acids research, 42(Database issue), pp.D490-5.
                    <br>
                    [7] Langille, M.G. et al., 2013. Predictive functional profiling of microbial communities using 16S rRNA marker gene sequences. Nat Biotechnol, 31(9), pp.814–821.
                    <br>
                    [8] Nguyen, N.H. et al., 2016. FUNGuild: An open annotation tool for parsing fungal community datasets by ecological guild. Fungal Ecology, 20, pp.241–248.
                  </p>
                </div>
                </div>
              </div>

								<div class="media">
								  <div class="media-left media-middle">
								    <a href="http://www.isme-microbes.org/isme16" target="_blank">
								      <img class="media-object" width="64" height="64" src="img/Logo_ISMEMontreal.png" alt="16th International Symposium on Microbial Ecology">
								    </a>
								  </div>
								  <div class="media-body">
								    <h4 class="media-heading">Study of microbial diversity and plant cell wall-degrading enzymes during flax dew-retting by using targeted-metagenomics and metatranscriptomics</h4>
									<h5><u>Christophe Djemiel</u>, Sébastien Grec, and Simon Hawkins</h5>
									<span class="label label-default">16th ISME</span>
									&nbsp;<span class="flag-icon flag-icon-ca" title="Montréal - Canada"></span>&nbsp;Montréal - Canada&nbsp;<i class="fas fa-calendar-check fa-fw" aria-hidden="true"></i>21<sup>th</sup> - 26<sup>th</sup> of August 2016&nbsp;<i class="fas fa-clock fa-fw" aria-hidden="true"></i> 3min
									&nbsp;<i class="fas fa-file-pdf fa-fw" aria-hidden="true"></i>&nbsp;<a href="#" target="_blank" class="text-orange">Proceedings</a>
									&nbsp;<a class="headingCV text-orange">abstract <i class="fas fa-caret-down" aria-hidden="true"></i></a>
									<div class="alert alert-success contentCV" role="alert">
									  <p class="text-justify">
									  The first step in the industrial transformation of flax stems into fibers is achieved by dew-retting on the soil surface (Tahir et al., 2011 and Akin et al., 2013). Despite many studies of this process relatively little is known about i) the composition and the evolution of the microflora population, ii) the kinetics of microbial colonization of plant material and iii) the composition and the evolution of the plant cell wall-degrading enzymes targeted to the different CAZy families (Lombard et al., 2014).
									  <br>
									  	In this work, we report the first exhaustive microbial inventory of dew retting. We conducted a targeted-metagenomics approach to identify for the first time the composition of the microbial communities (focusing on bacteria and fungi) involved in flax dew-retting. This approach has also allowed us to identify some potential major enzymatic functions related to cell wall degradation based on functional prediction using the bioinformatic software PICRUSt (Langille et al., 2013). 	In a second step we developed a metatranscriptomic approach to access the exogenous (soil) and endogenous (plant) enzymatic arsenal potentially involved in degrading cell wall polysaccharides during dew-retting in flax.
									  <br>
									  	The methodology used to explore microbial and enzymatic diversity using the Next Generation of Sequencing (Illumina system) will be described. We will then correlate colonization dynamics to plant cell wall degradation. Finally, the microbial ecology of the retting process will be compared to the degradation process of other plant-based material (e.g. forest litter).
									  <br>
									  This work is funded within the framework of the collaborative French ‘Future project’ SINFONI.
									  </p>
									</div>
								  </div>
								</div>
								<div class="media">
								  <div class="media-left media-middle">
								    <a href="http://jobim2016.sciencesconf.org/" target="_blank">
								      <img class="media-object" width="64" height="64" src="img/logo_jobim2016.jpg" alt="17th Journées Ouvertes en Biologie, Informatique et Mathématiques">
								    </a>
								  </div>
								  <div class="media-body">
								    <h4 class="media-heading">Study of microbial diversity and plant cell wall-degrading enzymes during flax dew-retting by using targeted-metagenomics</h4>
									<h5><u>Christophe Djemiel</u>, Sébastien Grec, and Simon Hawkins</h5>
									<span class="label label-default">17th JOBIM</span>
									&nbsp;<span class="flag-icon flag-icon-fr" title="Lyon - France"></span>&nbsp;Lyon - France&nbsp;<i class="fas fa-calendar-check fa-fw" aria-hidden="true"></i>28<sup>th</sup> - 30<sup>th</sup> of June 2016&nbsp;<i class="fas fa-file-pdf fa-fw" aria-hidden="true"></i>&nbsp;<a href="#" target="_blank" class="text-orange">Proceedings</a>
									&nbsp;<a class="headingCV text-orange">abstract <i class="fas fa-caret-down" aria-hidden="true"></i></a>
									<div class="alert alert-success contentCV" role="alert">
									  <p class="text-justify">
									  The first step in the industrial transformation of flax stems into industrial fibers used in textiles or composite materials is achieved by dew-retting on the soil surface. At harvest, the flax stems are ‘pulled’ (up-rooted) and laid down directly on the soil in swaths (strips). Subsequent exposure to alternating periods of rain and heat combined with the development and the action of soil microflora favor the separation of cellulosic bast fibers from the stems via the hydrolysis of cell wall polymers (hemicelluloses, pectins). Despite many studies of this process (Akin, 2013; Henriksson, & Akin, 1997; Martin et al., 2013) relatively little is known about i) the composition of the microflora population, ii) the kinetics of microbial colonization of plant material and iii) the evolution of the microbial population in the soil. As an example of biological processing involving microbial activities, dew retting represents a convenient model to study the microbial dynamics of plant colonization and degradation of cell wall components.
									  <br>
									  	In order to study this biological process, we undertook a targeted-metagenomics approach. A metagenomic pipeline including sample preparation, marker amplification, raw sequence and data analyses was built. We evaluated and compared different protocols and tools such as mothur (Schloss et al., 2009), qiime (Caporaso et al., 2010), PIPITS (Gweon et al., 2015) and clustering tools such as swarm (Mahé et al., 2014). The resulting pipeline analyses and homemade tools used to redesign primers so as to specifically amplify bacterial rRNA genes from complex samples including plants are detailed. Analyses are completed by functional predictions of enzymatic activity using an adaptation of PiCRUSt (Langille et al., 2013) software for the CAZy database (Lombard et al., 2014).
									  <br>
									  	As a conclusion and proof of pipeline efficiency, we report the first exhaustive microbial inventory of dew-retting and its evolution during the process obtained by using a targeted-metagenomics approach. Potential enzymatic functions related to cell wall degradation based on functional prediction using the bioinformatic software PICRUSt are listed and used to establish a potential chronology of cell wall polymer degradation.
									  <br>
									  This work is funded within the framework of the collaborative French ‘Future project’ SINFONI. Christophe Djemiel thanks the ‘Haut de France region’, and OSEO for their financial support. The authors thank the Genomic platform of Genopole Toulouse  Midi Pyrénées where sequencing was performed.
									  <br>
									  References :
									  <br>
									  Akin, D. E. (2013). Linen most useful: Perspectives on structure, chemistry, and enzymes for retting flax. ISRN Biotechnology, 	2013, 23. http://doi.org/http://dx.doi.org/10.5402/2013/186534
									  <br>
									  Henriksson, G., & Akin, D. (1997). Identification and retting efficiencies of fungi isolated from dew-retted flax in the United 	States and europe. Applied and Environmental Microbiology. Retrieved from http://aem.asm.org/content/63/10/3950.short
									  <br>
									  Martin, N., Mouret, N., Davies, P., & Baley, C. (2013). Influence of the degree of retting of flax fibers on the tensile properties of 	single fibers and short fiber/polypropylene composites. Industrial Crops and Products, 49, 755–767. http://doi.org/10.1016/j.indcrop.2013.06.012
									  <br>
									  Schloss, P. D., Westcott, S. L., Ryabin, T., Hall, J. R., Hartmann, M., Hollister, E. B., … Weber, C. F. (2009). Introducing mothur: open-source, platform-independent, community-supported software for describing and comparing microbial communities. Applied and Environmental Microbiology, 75(23), 7537–41. http://doi.org/10.1128/AEM.01541-09
									  <br>
									  Caporaso, J. G., Kuczynski, J., Stombaugh, J., Bittinger, K., Bushman, F. D., Costello, E. K., … Knight, R. (2010). QIIME allows analysis of high-throughput community sequencing data. Nature Publishing Group, 7(5), 335–336. http://doi.org/10.1038/nmeth0510-335
									  <br>
									  Gweon, H. S., Oliver, A., Taylor, J., Booth, T., Gibbs, M., Read, D. S., … Schonrogge, K. (2015). PIPITS: An automated pipeline for analyses of fungal internal transcribed spacer sequences from the Illumina sequencing platform. Methods in Ecology and Evolution, 6(8), 973–980. http://doi.org/10.1111/2041-210X.12399
									  <br>
									  Mahé, F., Rognes, T., Quince, C., de Vargas, C., & Dunthorn, M. (2014). Swarm: robust and fast clustering method for amplicon-based studies. PeerJ, 2, e593. http://doi.org/10.7717/peerj.593
									  <br>
									  Langille, M. G., Zaneveld, J., Caporaso, J. G., McDonald, D., Knights, D., Reyes, J. A., … Huttenhower, C. (2013). Predictive 	functional profiling of microbial communities using 16S rRNA marker gene sequences. Nat Biotechnol, 31(9), 814–821. 	http://doi.org/10.1038/nbt.2676
									  <br>
									  Lombard, V., Golaconda Ramulu, H., Drula, E., Coutinho, P. M., & Henrissat, B. (2014). The carbohydrate-active enzymes 	database (CAZy) in 2013. Nucleic Acids Research, 42(Database issue), D490–5. http://doi.org/10.1093/nar/gkt1178
									  </p>
									</div>
								  </div>
  								<div class="media">
  								  <div class="media-left media-middle">
  								    <a href="http://www.cwm2016.org/" target="_blank">
  								      <img class="media-object" width="64" height="64" src="img/logo_cellwall2016.jpg" alt="XIV Cell Wall Meeting">
  								    </a>
  								  </div>
  								  <div class="media-body">
  								    <h4 class="media-heading">Characterization of microbial diversity and plant cell wall-degrading enzymes during flax dew-retting by using targeted-metagenomics, metatranscriptomics and biochemistry</h4>
  									<h5>Christophe Djemiel, Sébastien Grec, and <u>Simon Hawkins</u></h5>
  									<span class="label label-default">XIV Cell Wall Meeting</span>
  									&nbsp;<span class="flag-icon flag-icon-gr" title="Chania - Greece"></span>&nbsp;Chania - Greece&nbsp;<i class="fas fa-calendar-check fa-fw" aria-hidden="true"></i>12<sup>th</sup> - 17<sup>th</sup> of August 2016&nbsp;<i class="fas fa-file-pdf fa-fw" aria-hidden="true"></i>&nbsp;<a href="#" target="_blank" class="text-orange">Proceedings</a>
  									&nbsp;<a class="headingCV text-orange">abstract <i class="fas fa-caret-down" aria-hidden="true"></i></a>
  									<div class="alert alert-success contentCV" role="alert">
  									  <p class="text-justify">
									  Flax (Linum usitatissimum L.) is an economically important oil and fiber crop. Flax bast fibers, rich in cellulose, are traditionally used to produce textiles (linen) and can also be used for reinforcing composite polymers as an environmentally friendly alternative to glass fibers. The ‘quality’ of both textiles and composite materials is influenced by the ‘quality’ of the plant fibers that arrive in the factory. Plant fiber quality itself is related to fiber cell wall structure determined during plant growth, and modified to a greater or lesser extent during fiber extraction.
									  <br>
									  	The first step in the industrial transformation of flax stems into fibers is achieved by dew-retting on the soil surface. At harvest, the flax stems are ‘pulled’ (up-rooted) and laid down directly on the soil in swaths (strips). Subsequent exposure to alternating periods of rain and heat combined with the development and the action of soil microflora favor the separation of cellulosic bast fibers from the stems via the hydrolysis of cell wall polymers (hemicelluloses, pectins). Despite many studies of this process relatively little is known about i) the composition and the evolution of the microflora population, ii) the kinetics of microbial colonization of plant material and iii) the composition and the evolution of the plant cell wall-degrading enzymes targeted to the different CAZy families.
									  <br>
									  	Dew-retting is not completely understood (e.g. only a few microbial species and enzymes have so far been identified) and our current lack of knowledge of certain aspects of this process represents a major obstacle to a more efficient retting process. In the framework of a collaborative French ‘Future project’ ‘StructuratIoN de la filière Fibres techniques d’OrigiNe végétale pour usages matérIaux’ (SINFONI) we are using a targeted-metagenomics approach to identify for the first time the composition of the microbial communities (focusing on bacteria and fungi) involved in flax dew-retting. This approach has also allowed us to identify some potential major enzymatic functions related to bacterial cell wall degradation based on functional prediction using the bioinformatic software PICRUSt. In a second phase aimed at confirming the PICRUSt prediction we developed a meta-transcriptomic approach to access the exogenous (soil) and endogenous (plant) enzymatic arsenal potentially involved in degrading cell wall polysaccharides during dew-retting in flax.
									  <br>
									  	In this poster communication, we will present the methodology used to explore microbial and enzymatic diversity using the Next Generation of Sequencing (Illumina Hiseq and MiSeq system) and our attempts to correlate this data with  morphological and chemical changes of the stem and enzymatic activities during retting. The identification of a range of biological and physico-chemical markers should contribute to the improvement of fiber extraction process and hence quality.
  									  </p>
  									</div>
  								  </div>
  								</div>
								<div class="media">
								  <div class="media-left media-middle">
								    <a href="http://www.polytech-lille.fr/nature-sciences-technologies-p7199.html#.V11nHuaLTeQ" target="_blank">
								      <img class="media-object" width="64" height="64" src="img/logo_ligno_polytech2016.png" alt="Matériaux biosourcés, biocomposites : Nature, Sciences & Technologies">
								    </a>
								  </div>
								  <div class="media-body">
								    <h4 class="media-heading">Meta-omics for a better understanding of the early stages of flax fiber extraction during retting</h4>
									<h5><u>C. Djemiel</u>, A. Portelette, B. Chabbert, B. Kurek, S. Grec, and S. Hawkins</h5>
									<span class="label label-default">Polytech Seminar</span>
									&nbsp;<span class="flag-icon flag-icon-fr" title="Lille - France"></span>&nbsp;Lille - France&nbsp;<i class="fas fa-calendar-check fa-fw" aria-hidden="true"></i>24<sup>th</sup> of March 2016&nbsp;<a class="headingCV text-orange">abstract <i class="fas fa-caret-down" aria-hidden="true"></i></a>
									<div class="alert alert-success contentCV" role="alert">
									  <p class="text-justify">
										  Coming soon ...
									  </p>
									</div>
								  </div>
								</div>
								<div class="media">
								  <div class="media-left media-middle">
								    <a href="https://www.u-picardie.fr/evenement/site_rfp/index.html#!prettyPhoto" target="_blank">
								      <img class="media-object" width="64" height="64" src="img/logo_RFP.jpg" alt="10e Journées du Réseau Français des Parois">
								    </a>
								  </div>
								  <div class="media-body">
								    <h4 class="media-heading">Étude de la dégradation pariétale au cours du rouissage du lin</h4>
									<h5><u>Christophe Djemiel</u>, Sébastien Grec, and Simon Hawkins</h5>
									<span class="label label-default">10th RFP</span>
									&nbsp;<span class="flag-icon flag-icon-fr" title="Amiens - France"></span>&nbsp;Amiens - France&nbsp;<i class="fas fa-calendar-check fa-fw" aria-hidden="true"></i>7<sup>th</sup> - 9<sup>th</sup> of July 2014&nbsp;<i class="fas fa-file-pdf fa-fw" aria-hidden="true"></i>&nbsp;<a href="https://www.u-picardie.fr/evenement/site_rfp/docs/livret.pdf" target="_blank" class="text-orange">Proceedings</a>
									&nbsp;<a class="headingCV text-orange">abstract <i class="fas fa-caret-down" aria-hidden="true"></i></a>
									<div class="alert alert-success contentCV" role="alert">
									  <p class="text-justify">
									  La culture du lin (Linum usitatissimum L.) huile et fibres est économiquement très importante. L’huile
									  extraite des graines de lin est considérée comme une source riche en acide gras omega-3, acide α-
									  linolénique (ALA), mais contient aussi des lignanes biologiquement actives ayant des effets
									  bénéfiques sur la santé humaine. Les plantes de lin sont également riches en fibres cellulosiques (Bast
									  fibers) traditionnellement utilisées dans l’industrie du textile. Ces fibres peuvent aussi être utilisées
									  dans le renforcement de polymères composites ou comme substitut écologique pour les fibres de verre
									  [1]. La qualité des textiles et des matériaux composites est directement impactée par la qualité des
									  fibres végétales arrivant à l’usine. La qualité de la fibre est-elle même influencée au cours de la
									  croissance de la plante mais aussi lors de l’extraction de la fibre, correspondant à la première étape
									  dans la transformation industrielle des fibres végétales.
									  <br>
									  Cette première étape de transformation se déroule au cours du rouissage sur champ. Lors de la récolte
									  les tiges sont arrachées (avec les racines) et sont directement déposées sur le sol en andains.
									  L’alternance de pluies et de chaleurs combinées au développement et à l’action des microorganismes
									  du sol favorise la séparation des fibres cellulosiques des tiges grâce à l’hydrolyse des polymères de la
									  paroi cellulaire (hémicelluloses, pectines…). Malgré plusieurs études de ce procédé [2,3,4], on sait
									  relativement peu de chose sur i) la composition des populations de la microflore, ii) la cinétique de la
									  colonisation microbienne de la matière végétale, iii) l’évolution de la population microbienne du sol.
									  Le rouissage sur champ reste une ‘boite noire’, et notre manque actuel de connaissance de ce
									  processus représente un obstacle majeur pour un rouissage plus efficace.
									  <br>
									  Dans le cadre d’un projet français, visant à la ‘ StructuratIoN de la filière Fibres techniques d'OrigiNe
									  végétale pour usages matérIaux’ (SINFONI), nous utiliserons une approche de “Méta-Barcoding” afin
									  d'entreprendre le premier inventaire complet ainsi qu’une étude cinétique des populations
									  microbiennes impliquées dans le rouissage du lin. Par la suite, une approche de méta-génomique
									  fonctionnelle nous permettra également d'identifier les principales fonctions enzymatiques liées à la
									  dégradation de la paroi cellulaire.
									  <br>
									  Cette étude représente la première caractérisation taxonomique et fonctionnelle exhaustive du
									  processus de rouissage du lin et nous permettra d'identifier une gamme de marqueurs biologiques qui
									  pourraient être utilisées pour améliorer le processus d'extraction et donc la qualité de la fibre.
									  </p>
									</div>
								  </div>
								</div>
								<div class="media">
								  <div class="media-left media-middle">
								    <a href="http://ge-lyon2014.sciencesconf.org/" target="_blank">
								      <img class="media-object" width="64" height="64" src="img/logo_bge2014.png" alt="Bioinformatics for Environmental Genomics">
								    </a>
								  </div>
								  <div class="media-body">
								    <h4 class="media-heading">“Like looking for a needle in a flax stack”</h4>
									<h5><u>Christophe Djemiel</u>, Sébastien Grec, and Simon Hawkins</h5>
									<span class="label label-default">Bioinformatics for Environmental Genomics</span>
									&nbsp;<span class="flag-icon flag-icon-fr" title="Lyon - France"></span>&nbsp;Lyon - France&nbsp;<i class="fas fa-calendar-check fa-fw" aria-hidden="true"></i>27<sup>th</sup> - 28<sup>th</sup> of May 2014&nbsp;<i class="fas fa-file-pdf fa-fw" aria-hidden="true"></i>&nbsp;<a href="http://ge-lyon2014.sciencesconf.org/conference/ge-lyon2014/Abstracts.pdf" target="_blank" class="text-orange">Proceedings</a>
									&nbsp;<a class="headingCV text-orange">abstract <i class="fas fa-caret-down" aria-hidden="true"></i></a>
									<div class="alert alert-success contentCV" role="alert">
									  <p class="text-justify">
									  Flax (Linum usitatissimum L.) is an economically important oil and fiber crop. Oil extracted from flax seeds (linseed) is considered as a rich source of the omega-3 fatty acid, a-linolenic acid (ALA) and seeds also contain biologically active lignans that have beneficial effects on human health. Flax plants are also the source of cellulose-rich bast fibers that have traditionally been used for textiles (linen). These fibers can also be used for reinforcing composite polymers as an environmentally-friendly substitute for glass fibers (Baley, 2002). The quality of both textiles and composite materials are affected by the quality of plant fibers arriving in the factory. Plant fiber quality itself is determined during plant growth, but is also modified during fiber extraction that represents the first step in the industrial transformation of plant fibers.
									  <br>
									   The first step in the transformation of flax stems into fibers involves dew retting on the soil surface. At harvest the flax stems are pulled (up-rooted) and laid down directly on the soil in swaths (strips). Subsequent exposure to alternating periods of rain and heat combined with the development and the action of the soil microflora favor the separation of cellulosic bast fibers from the stems via the hydrolysis of cell wall polymers (hemicelluloses, pectins). Despite several studies of this process (Akin, 2013; Martin et al., 2013;Henriksson et al., 1997), relatively little is known about i) the composition of the microflora population, ii) the kinetics of microbial colonization of plant material, ii) the evolution of the microbial population in the soil. Soil retting remains a “black box”, and our current lack of knowledge of this process represents a major obstacle to a more efficient retting process. In the framework of a collaborative French “Future project” "StructuratIoN de la filière Fibres techniques d'OrigiNe végétale pour usages matérIaux" (SINFONI), we are using a metagenomics approach to undertake the first complete inventory and kinetic study of the microbial populations involved in flax retting. Furthermore, this approach will also allow us to identify the major enzymatic functions related to cell wall degradation.
									  <br>
									  This study represents the first exhaustive taxonomic and functional characterization of the flax retting process and will allow us to identify a range of biological markers what could be used to improve the extraction process and hence fiber quality.
									  <br>
									  References:
									  <br>
									  Martin et al., (2013). Influence of the degree of retting of flax fibers on the tensile properties of single fibers and short fiber/polypropylene composites. Industrial Crops and Products, 49, 755–767. doi:10.1016/j.indcrop.2013.06.012
									  <br>
									  D.E. Akin, Linen most useful: perspectives on structure, chemistry, and enzymes for retting flax, ISRN Biotechnol. 2013 (2013) 23 (Article ID 186534) http://dx.doi.org/10.5402/2013/186534.
									  <br>
									  Baley C: Analysis of the flax fibres tensile behaviour and analysis of the tensile stiffness increase. Composites Part A: Applied Science and Manufacturing 2002, 33:939‚Äì948.
									  <br>
									  Henriksson et al., (1997). Identification and retting efficiencies of fungi isolated from dew-retted flax in the United States and europe. Applied and Environmental Microbiology, 63(10), 3950–6.
									  </p>
									</div>
								  </div>
								</div>
								<div class="media">
								  <div class="media-left media-middle">
								    <a href="https://colloque6.inra.fr/jobim2013" target="_blank">
								      <img class="media-object" width="64" height="64" src="img/logo_jobim2013.png" alt="Journées Ouvertes en Biologie, Informatique et Mathématiques">
								    </a>
								  </div>
								  <div class="media-body">
								    <h4 class="media-heading">Integrated next generation sequencing storage and processing environment</h4>
									<h5><u>J. Mariette</u>, F. Escudie, A. Leleu, C. Djemiel, P. Bardou, C. Kushly, G.Salin, C. Hoede, C. Noirot and C. Klopp</h5>
									<span class="label label-default">14th JOBIM</span>
									&nbsp;<span class="flag-icon flag-icon-fr" title="Toulouse - France"></span>&nbsp;Toulouse - France&nbsp;<i class="fas fa-calendar-check fa-fw" aria-hidden="true"></i>1<sup>th</sup> - 4<sup>th</sup> of July 2013&nbsp;<i class="fas fa-file-pdf fa-fw" aria-hidden="true"></i>&nbsp;<a href="pdf/2013_Jobim_Vol2_Résumés_courts_poster_toulouse.pdf" target="_blank" class="text-orange">Proceedings</a>
									&nbsp;<a class="headingCV text-orange">abstract <i class="fas fa-caret-down" aria-hidden="true"></i></a>
									<div class="alert alert-success contentCV" role="alert">
									  <p class="text-justify">
									  1 Background
									  <br>
									  Next generation sequencing platforms are now well implanted in sequencing centres and some
									  laboratories. Upcoming smaller scale machines such as the 454 junior from Roche or the MiSeq from
									  Illumina will increase the number of laboratories hosting a sequencer. In such a context, it is important to
									  provide these teams with an easily manageable environment to store and process the produced reads.
									  NG6 [1] is an information system providing a set of automated analysis pipelines built to process NGS
									  (Next Generation Sequencing) data which can be executed locally or in a cluster environment. The v2.0 is
									  built upon the jflow [2] workflow management system.
									  <br>
									  2 Implementation
									  <br>
									  NG6 offers three user statuses associated to a run (administrator, manager and member) and two data
									  access levels: public and private.
									  If the connected user is a project
									  administrator, he will be able to run available
									  workflows within the graphical user interface.
									  Once the pipeline execution is over, all newly
									  added analysis and runs will be flagged as
									  hidden. This is meant to permit the validation of
									  the run by the team in charge of the sequencer
									  before data release to manager. When the data
									  are released, the manager is then allowed to give
									  access to project members.
									  The connected user will be able to
									  download raw and processed data and browse
									  through the analysis results, statistics and
									  parameters.
									  NG6 includes a workflow environment already containing pipelines adapted to different input formats
									  (sff, fasta, fastq and qseq), different sequencers (roche 454, Illumina hiseq and miseq) and various analyses :
									  quality check (casava, illumina, roche 454), rnaseq (hiseq) and diversity (miseq, roche 454).
									  <br>
									  3 Extending NG6
									  <br>
									  Adding new analysis component into NG6 requires two steps. The first one is writing the jflow
									  component of the analysis using the NG6 API to define the data stored in the database and the result files
									  stored in the directory structure. Second, a smarty [3] template is specified to set the corresponding analysis
									  display. While writing the smarty template, the developer has access to several objects to build the analysis
									  display as wished.
									  Moreover, NG6 can easily be extended to handle data outside of the NGS world such as metabolomic
									  data which should be the next kind of data handled by the system.
									  <br>
									  References
									  <br>
									  [1] Mariette J, Escudie F, Allias N, Salin G, Noirot C, Thomas S, Klopp C. NG6: Integrated next generation
									  sequencing storage and processing environment. BMC Genomics 2012, 13:462.
									  <br>
									  [2] https://mulcyber.toulouse.inra.fr/plugins/mediawiki/wiki/jflow/index.php/Accueil
									  <br>
									  [3] http://www.smarty.net/
									  </p>
									</div>
								  </div>
								</div>
								</div>
							</div>
						</div>
					</div>
				</div>
				<div class="row">
					<div class="col-xs-12 col-sm-12 col-md-12">
						<div class="row">
							<div class="col-xs-4 col-sm-4 col-md-4">
								<h2>Computer skills</h2>
								<hr>
								<div class="row">
									<div class="col-xs-12 col-sm-12 col-md-12">
										<div class="row">
											<div class="col-xs-6 col-sm-6 col-md-6">
												<h6><strong class="text-muted">Operating system</strong></h6>
												<h6>GNU/Linux, macOS</h6>
												<h6><strong class="text-muted">OOP</strong></h6>
												<h6>Python, Java</h6>
												<h6><strong class="text-muted">Programming languages</strong></h6>
												<h6>Perl, Shell, LISP, Hoc</h6>
												<h6><strong class="text-muted">Office software</strong></h6>
												<h6><span class="latex">L<sup>A</sup>T<sub>E</sub>X</span>/Beamer, LibreOffice, Adobe Illustrator</h6>
											</div>
											<div class="col-xs-6 col-sm-6 col-md-6">
												<h6><strong class="text-muted">IDE, collaboration</strong></h6>
												<h6>Eclipse, SVN, github</h6>
												<h6><strong class="text-muted">Web</strong></h6>
												<h6>HTML/CSS, jQuey, PHP</h6>
												<h6><strong class="text-muted">Database</strong></h6>
												<h6>MySQL, PostgreSQL</h6>
											</div>
										</div>
									</div>
								</div>
							</div>
							<div class="col-xs-4 col-sm-4 col-md-4">
								<h2>Data analysis</h2>
								<hr>
								<div class="row">
									<div class="col-xs-12 col-sm-12 col-md-12">
										<h6><strong class="text-muted">Workflow management system</strong></h6>
										<h6><a href="http://genoweb.toulouse.inra.fr:8090/app/index.html" target="_blank" class="text-orange">Jflow</a></h6>
										<h6><strong class="text-muted">Diversity tools</strong></h6>
										<h6><a href="http://www.mothur.org/wiki/Main_Page" target="_blank" class="text-orange">Mothur</a>, <a href="https://github.com/hsgweon/pipits" target="_blank" class="text-orange">PIPTIS</a>, <a href="http://picrust.github.io/picrust/" target="_blank" class="text-orange">Picrust</a>, <a href="https://bitbucket.org/nsegata/lefse" target="_blank" class="text-orange">LEfSe</a>, <a href="http://merenlab.org/software/oligotyping/" target="_blank" class="text-orange">Oligotyping</a>, <a href="https://github.com/UMNFuN/FUNGuild" target="_blank" class="text-orange">FUNGuild</a></h6>
										<h6><strong class="text-muted">Statistical</strong></h6>
										<h6>Diversity measures, Multivariate and Multiple comparison analysis with R</h6>
										<h6><strong class="text-muted">Visualization tools</strong></h6>
										<h6><a href="http://www.highcharts.com/" target="_blank" class="text-orange">Highcharts</a>, <a href="https://github.com/marbl/Krona/wiki" target="_blank" class="text-orange">Krona</a>, <a href="http://bioinfo.genotoul.fr/jvenn/index.html" target="_blank" class="text-orange">jvenn</a></h6>
									</div>
								</div>
							</div>
							<div class="col-xs-4 col-sm-4 col-md-4">
								<h2>Biology skills</h2>
								<hr>
								<div class="row">
									<div class="col-xs-12 col-sm-12 col-md-12">
										<h6><strong class="text-muted">Methodology</strong></h6>
										<h6>Sampling, storage</h6>
										<h6><strong class="text-muted">Molecular technics</strong></h6>
										<h6>RNA/DNA Extraction, Purification, Quantification by Picogreen and LightCycler, Normalization by Epmotion, PCR, Cloning</h6>
									</div>
								</div>
							</div>
						</div>
					</div>
				</div>
				<div class="row">
					<div class="col-xs-12 col-sm-12 col-md-12">
						<h2>Referees</h2>
						<hr>
						<div class="row">
							<div class="col-xs-4 col-sm-4 col-md-4">
								<p>
									<strong>Pr. Simon Hawkins</strong>
									<br>
									My Ph.D. supervisor
									<br>
									<i class="fa fa-briefcase fa-fw" aria-hidden="true"></i>&nbsp;Plant Fiber Team leader
									<br>
									<i class="fa fa-university fa-fw" aria-hidden="true"></i><a href="http://ugsf-umr-glycobiologie.univ-lille1.fr/" target="_blank" class="text-orange">&nbsp;UMR 8576 CNRS - Lille1 - UGSF</a>
									<br>
									<i class="fa fa-envelope fa-fw" aria-hidden="true"></i>&nbsp;first.last@univ-lille1.fr
									<br>
									<i class="fa fa-phone fa-fw" aria-hidden="true"></i>&nbsp;+33&nbsp;(0)3&nbsp;20&nbsp;43&nbsp;40&nbsp;30
								</p>
							</div>
							<div class="col-xs-4 col-sm-4 col-md-4">
								<p>
									<strong>Dr. Sebastien Grec</strong>
									<br>
									My Ph.D. supervisor
									<br>
									<i class="fa fa-briefcase fa-fw" aria-hidden="true"></i>&nbsp;Plant Fiber Team assitant professor
									<br>
									<i class="fa fa-university fa-fw" aria-hidden="true"></i><a href="http://ugsf-umr-glycobiologie.univ-lille1.fr/" target="_blank" class="text-orange">&nbsp;UMR 8576 CNRS - Lille1 - UGSF</a>
									<br>
									<i class="fa fa-envelope fa-fw" aria-hidden="true"></i>&nbsp;first.last@univ-lille1.fr
									<br>
									<i class="fa fa-phone fa-fw" aria-hidden="true"></i>&nbsp;+33&nbsp;(0)3&nbsp;20&nbsp;33&nbsp;72&nbsp;51
								</p>
							</div>
							<div class="col-xs-4 col-sm-4 col-md-4">
								<p>
									<strong>Jerome Mariette (software engineer)</strong>
									<br>
									My M.Sc. internship supervisor
									<br>
									<i class="fa fa-briefcase fa-fw" aria-hidden="true"></i>&nbsp;GenoToul Bioinfo Team
									<br>
									<i class="fa fa-university fa-fw" aria-hidden="true"></i><a href="http://bioinfo.genotoul.fr/index.php?id=4" target="_blank" class="text-orange">&nbsp;INRA MIA-T - Borde Rouge</a>
									<br>
									<i class="fa fa-envelope fa-fw" aria-hidden="true"></i>&nbsp;first.last@toulouse.inra.fr
									<br>
									<i class="fa fa-phone fa-fw" aria-hidden="true"></i>+33&nbsp;(0)5&nbsp;61&nbsp;28&nbsp;57&nbsp;25
								</p>
							</div>
						</div>
					</div>
				</div>
			</div>
			<div class="col-xs-3 col-sm-3 col-md-3">
        <br>
        <img src="img/christophe_djemiel_desk_sn2.png" alt="christophe_djemiel_desk_sn2" class="center-block img-rounded img-responsive">
				<br>
        <p><strong>Christophe DJEMIEL, Ph.D. </strong><a href=" https://orcid.org/0000-0002-5659-7876" target="_blank"><img src='img/orcid.png' alt="0000-0002-5659-7876" class="img-responsive" style="display:inline; white-space:nowrap; max-height: 15px; max-width: 15px;"></a></p>
				<p><i class="fas fa-file-pdf fa-fw" aria-hidden="true"></i>&nbsp;<a href="pdf/2016-christophe_djemiel-cv.pdf" target="_blank" class="text-orange">curriculum vitae (v06.2016)</a></p>
				<!-- <p><i class="fa fa-university fa-fw" aria-hidden="true"></i>&nbsp;<a href="http://ugsf-umr-glycobiologie.univ-lille1.fr/" target="_blank" class="text-orange">University of Lille 1 - UGSF (Fr)</a></p> -->
        <p><i class="fas fa-briefcase fa-fw" aria-hidden="true"></i>&nbsp;big data | ecology | bioinformatics</p>
				<p><i class="fas fa-envelope fa-fw" aria-hidden="true"></i>&nbsp;first.last@gmail.com</p>
				<p><i class="fas fa-map-marker fa-fw" aria-hidden="true"></i>&nbsp;Lille, France</p>
			</div>
		</div>
    </div> <!-- /container -->

    <footer class="footer">
      <div class="container">
        <p class="text-muted"><i class="fa fa-copyright fa-fw" aria-hidden="true"></i>Powered by <a href="http://getbootstrap.com/" target="_blank" class="text-orange"> bootstrap v3.3.6</a> | Christophe DJEMIEL | 2018 | Last updated: 2018.03.01</p>
      </div>
    </footer>
    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="js/jquery-3.0.0.min.js"></script>
    <script>window.jQuery || document.write('<script src="js/vendor/jquery.min.js"><\/script>')</script>
    <script src="js/bootstrap.min.js"></script>
    <!-- IE10 viewport hack for Surface/desktop Windows 8 bug -->
    <script src="js/ie10-viewport-bug-workaround.js"></script>
    <script src="js/affix.js"></script>
    <script src="js/dropdown.js"></script>
    <script src="js/scholar-0.1.1.min.js"></script>
	  <script type="text/javascript">
	  $(document).ready(function() {
	  $(".contentCV").hide();
  	$(".headingCV").show();
	  //toggle the componenet with class msg_body
	  $(".headingCV").click(function()
	  {
	     $(this).next(".contentCV").slideToggle(500);
	  });
    Scholar.load("Nm3Y4T0AAAAJ");
	  });
	</script>
  </body>
</html>
